cmd/web/patient_signup_form.templ

```diff
@@ -126,6 +126,17 @@ templ PatientSignupScreen() {
                   <p class="mt-2 text-sm text-gray-500">Your password must be at least 8 characters long.</p>
                 </div>
 
+                <div class="sm:col-span-4 mt-4">
+                  <label for="confirm_password" class="block text-sm font-medium leading-6 text-gray-900">
+                    Re-enter Password <span class="text-red-600">*</span>
+                  </label>
+                  <div class="mt-2">
+                    <input id="confirm_password" type="password" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6" placeholder="Enter your password again">
+                  </div>
+                </div>
+
+                <p id="password-error" class="text-sm text-red-600 hidden">Passwords do not match</p>
```

Comment:
This is displayed only when passwords do not match. I've added a js file checking if they match or not and then makes this visible.

Reply:
yes I have checked the functionality and it is working. I was more referring to the way it is displayed in the UI. It can be ignored as of now. I am planning to revisit the validation logic implemented through parsley package.